spec/factories/person.rb

```diff
@@ -12,4 +23,9 @@
       end
     end
   end
+
+  factory :person_without_profiles, class: 'Person' do
```

Comment:
don't think we need to define the class here anymore as it's under the correct factory?

Reply:
I'm actually not sure whether the symbol or the file are used to know the class :). I'll have a play

Reply:
Uses the symbol, sadly :(. Could do with refactoring our factories a bunch I think.

Reply:
ah your right, somehow in my head this was a trait not a factory, as you were :D